src/Credentials.ts

```diff
@@ -838,10 +877,35 @@ class Credentials {
    *  @return   {Promise<Object, Error>}                        a promise which resolves with a parsed response or rejects with an error.
    */
   async verifyDisclosure(token: string) {
-    const { payload, doc } = await verifyJWT(token, { audience: this.did })
+    const { payload, doc } = await verifyJWT(token, { resolver: this.resolver, audience: this.did })
     return this.processDisclosurePayload({ payload, doc })
   }
 
+  createPresentationRequest(params: PresentationRequestParams, expiresIn = 600) {
+    const payload: PresentationRequest = {}
+    if (params.iss) payload.iss = params.iss
+    if (params.claims) payload.claims = params.claims
+    if (params.callbackUrl) payload.callback = params.callbackUrl
+
+
+    return this.signJWT(
+      { ...payload, type: Types.PRESENTATION_REQUEST },
+      expiresIn ? undefined : expiresIn
```

Comment:
If I'm reading this right, then expiresIn never gets passed in unless it's "falsey" which is probably not the intent